### DISCUSSION TITLE
Can a revocation certificate for an `OpenPGPPlainSigner` worker be generated post-factum?

### DISCUSSION THREAD

**Discussion by ralienpp** [Keyfactor/signserver-ce#43] [Q&A] [answered]
I have a worker of the type OpenPGPPlainSigner, it was created with `GENERATE_REVOCATION_CERTIFICATE=FALSE`. Is it possible to generate such a revocation certificate after the worker was successfully created and used for producing signatures?

Such functionality is available in the `gpg` client itself, but I did not find a way to use it via the web-ui.

**Comment by netmackan** [comment, marked as answer]
Yes, if I recall correctly you should be able to do [Step 3](https://doc.primekey.com/signserver/signserver-operations/setting-up-openpgp-signer#SettingupOpenPGPSigner-Step3:GenerateandStoreRevocationCertificate) later as well. The only important thing is that the property is set to true while you do "Generate CSR" if you want it to return the revocation certificate and to have it set to false when you want to add certification.
